src/components/WebexActivity/WebexActivity.stories.js

```diff
@@ -8,139 +8,50 @@ import {activities, people} from '../../data';
-      peopleAdapter: new PeopleJSONAdapter(people),
-    }}
-  />
-));
-
-stories.add('no activity header', () => {
-  newActivities[activityID] = {
-    ...activities[activityID],
-    displayHeader: false,
-  };
-
-  return (
-    <WebexActivity
-      activityID={activityID}
-      adapters={{
-        activitiesAdapter: new ActivitiesJSONAdapter(newActivities),
-        peopleAdapter: new PeopleJSONAdapter(people),
-      }}
-    />
-  );
-});
+stories.add('default', () => <WebexActivity activityID="default" adapters={adapters} />);
 
-stories.add('multi-line text', () => {
-  newActivities[activityID] = {
-    ...activities[activityID],
-    text:
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.\nUt enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.\nDuis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur.\nExcepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
-  };
-
-  return (
-    <WebexActivity
-      activityID={activityID}
-      adapters={{
-        activitiesAdapter: new ActivitiesJSONAdapter(newActivities),
-        peopleAdapter: new PeopleJSONAdapter(people),
-      }}
-    />
-  );
-});
+stories.add('no activity header', () => <WebexActivity activityID="no-header" adapters={adapters} />);
 
-stories.add('long text', () => {
-  newActivities[activityID] = {
-    ...activities[activityID],
-    text:
-      'Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat. Duis aute irure dolor in reprehenderit in voluptate velit esse cillum dolore eu fugiat nulla pariatur. Excepteur sint occaecat cupidatat non proident, sunt in culpa qui officia deserunt mollit anim id est laborum.',
-  };
-
-  return (
-    <WebexActivity
-      activityID={activityID}
-      adapters={{
-        activitiesAdapter: new ActivitiesJSONAdapter(newActivities),
-        peopleAdapter: new PeopleJSONAdapter(people),
-      }}
-    />
-  );
-});
+stories.add('multi-line text', () => <WebexActivity activityID="multi-line" adapters={adapters} />);
+
+stories.add('long text', () => <WebexActivity activityID="long" adapters={adapters} />);
 
 stories.add('created today', () => {
   const today = new Date().toString();
 
-  newActivities[activityID] = {...activities[activityID], text: `${today}`, created: today};
-
-  return (
-    <WebexActivity
-      activityID={activityID}
-      adapters={{
-        activitiesAdapter: new ActivitiesJSONAdapter(newActivities),
-        peopleAdapter: new PeopleJSONAdapter(people),
-      }}
-    />
-  );
+  activities.today = {...activities.today, text: `${today}`, created: today};
```

Comment:
don't you want to update the `created` field instead of the text?

Reply:
addressed